lib/driver.js

```diff
@@ -287,6 +293,12 @@ class AndroidDriver extends BaseDriver {
     await this.bootstrap.sendAction("compressedLayoutHierarchy", {compressLayout: compress});
   }
 
+  // Set UiAutomator Configurator related configurations
+  // value sould be like {"method":"setWaitForIdleTimeout","value":5000}
+  async setConfiguratorConfig (value) {
+    await this.bootstrap.sendAction("configurator", JSON.parse(value));
```

Comment:
I think it might be best to also check that it is a String before trying to parse it. I can see problems if an actual JSON object (rather than a stringified version) was passed in.